### PR TITLE
Verify that TCP ports in network test

### PR DESCRIPTION
**Pull request by ejj**

### CODE REVIEW

**Comment by luise** [comment]
@ejj , can you fix the PR title?

File: integration-tester/tests/10-network/connection_test.go [hunk 1]
```diff
@@ -55,6 +55,61 @@ func testPing(t *testing.T, sshUtil util.SSHUtil, containers []db.Container,
 	wg.Wait()
```
Comment (kklin) [inline integration-tester/tests/10-network/connection_test.go]: We decided that the prefix for integration test changes is `ci:`

**Comment by ejj** [comment]
@luise updated

File: integration-tester/tests/10-network/connection_test.go [hunk 2]
```diff
@@ -55,6 +55,61 @@ func testPing(t *testing.T, sshUtil util.SSHUtil, containers []db.Container,
...
+	for _, conn := range connections {
+		for _, from := range conn.From {
+			container, ok := containerMap[from]
+			if !ok {
+				t.Errorf("Unknown container: %s", from)
+				continue
+			}
+
+			for _, to := range conn.To {
+				for i := conn.MinPort; i <= conn.MaxPort; i++ {
```
Comment (luise) [inline integration-tester/tests/10-network/connection_test.go]: i --> port?

Comment (ejj) [reply integration-tester/tests/10-network/connection_test.go]: 👍

**Pull request merged**